modules/minimum-required-permissions-ipi-gcp-provided-sas.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="minimum-required-permissions-ipi-gcp-provided-sas_{context}"]
+= Required GCP permissions for user-provided service accounts
+
+When you are installing a cluster, the compute and control plane nodes require their own service accounts.
+By default, the installation program creates a service account for the control plane and compute nodes.
+The service account that the installation program uses requires the roles and permissions that are listed in the _Creating a service account in GCP_ section, as well as the `resourcemanager.projects.getIamPolicy` and `resourcemanager.projects.setIamPolicy` permissions.
+These permissions should be applied to the service account in the host project.
+If this approach does not meet the security requirements of your organization, you can provide a service account email address for the control plane or compute nodes in the `install-config.yaml` file.
+For more information, see the _Installation configuration parameters for GCP_ page.
+If you provide a service account for control plane nodes, you must grant that service account the `roles/compute.networkUser` role in the host project.
```

Comment:
It seems (but not sure) like this appears on all gcp installs, but this is only relevant for XPN installs.